VideoLocker/res/values-v19/styles.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="AppTheme.NoActionBar.TranslucentStatusBar">
+        <item name="android:windowTranslucentStatus">true</item>
```

Comment:
whoops, this is close, but not quite what we want. Move this file to `values-v21` instead, and set these values:
```xml
        <item name="android:windowDrawsSystemBarBackgrounds">true</item>
        <item name="android:statusBarColor">@android:color/transparent</item>
        <item name="android:windowTranslucentStatus">false</item>
```

This should fix the issue where the color is not quite right, and is only supported on API 21+. Older APIs will have a black status bar, this is normal :)